dashboard/lib/dynamic_config/gatekeeper.rb

```diff
@@ -0,0 +1,81 @@
+# A module that allows us to dynamically turn on and off
+# features without requiring a code push.
+
+require 'dynamic_config/datastore_cache'
+require 'dynamic_config/adapters/dynamodb_adapter'
+require 'dynamic_config/adapters/json_file_adapter'
+require 'dynamic_config/adapters/memory_adapter'
+
+class GatekeeperBase
```

Comment:
Is Gatekeeper the best name for this concept?  (DynamicConfig or something like it seems more likely.)